spec.bs

```diff
@@ -1145,38 +1148,45 @@ steps. They return an [=aggregation coordinator=], null or a {{DOMException}}:
 1. Return |origin|.
 
 To <dfn algorithm>obtain the pre-specified report parameters</dfn> given a
-{{SharedStorageRunOperationMethodOptions}} |options|, perform the following
-steps. They return a [=pre-specified report parameters=], null, or a
+{{SharedStorageRunOperationMethodOptions}} |options| and a [=Window/browsing context=] |context|,
```

Comment:
We might want to link just to `[=browsing context=]` (i.e. the type, not the field on window)

Reply:
Changed it to the browsing context type.